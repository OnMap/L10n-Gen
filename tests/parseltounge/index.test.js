const { uploadKeys } = require('../../src/parseltounge');

describe('Update Keys', () => {
  it('upload all keys', async () => {
    const token = 'test_token';
    const result = await uploadKeys(['testKey1', 'testKey2'], 'asd', token, 'dev');
    expect(result).toMatchSnapshot();
  });
});

