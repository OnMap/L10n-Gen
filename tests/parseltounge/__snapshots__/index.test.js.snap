// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Update Keys pass all keys 1`] = ['testKey1', 'testKey12'];
